docs/core/diagnostics/distributed-tracing-instrumentation-walkthroughs.md

```diff
@@ -29,7 +29,7 @@ dotnet new console
 
 Applications that target .NET 5 and later already have the necessary distributed tracing APIs included. For apps targeting older
 .NET versions, add the [System.Diagnostics.DiagnosticSource NuGet package](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/)
-version 5 or greater.
+version 5 or greater. For libraries targetting netstandard, we recommend referencing the latest version of the NuGet package.
```

Comment:
```suggestion
version 5 or greater. For libraries targetting netstandard, we recommend referencing the oldest LTS version of the package which is still supported and contains the APIs your library needs.
```
Does that work? If not feel free to use the suggestion feature and propose the exact text you want.

Reply:
I am afraid this does not cover the case if the required feature is in non-LTS. Implicitly telling don't use it. My comment https://github.com/dotnet/docs/pull/43269#discussion_r1825175898 may cover all cases? I am not suggesting taking it as it is. Just cover the cases we need to talk about.

Reply:
Edited to remove the LTS, how about now?